src/Symfony/Component/Ldap/Adapter/ExtLdap/UpdateOperation.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Adapter\ExtLdap;
+
+use Symfony\Component\Ldap\Exception\UpdateOperationException;
+
+class UpdateOperation
+{
+    private $operationType;
+    private $values;
+    private $attribute;
+
+    private $validOperationTypes = array(
+        LDAP_MODIFY_BATCH_ADD,
+        LDAP_MODIFY_BATCH_REMOVE,
+        LDAP_MODIFY_BATCH_REMOVE_ALL,
+        LDAP_MODIFY_BATCH_REPLACE,
+    );
+
+    /**
+     * @param int        $operationType An LDAP_MODIFY_BATCH_* constant
+     * @param string     $attribute     The attribute to batch modify on
+     * @param array|null $values
```

Comment:
This `@param` can be removed (as it does not add anything that is not already in the signature)